tests/src/test/scala/org/json4s/reflect/DescriptorsSpec.scala

```diff
@@ -0,0 +1,116 @@
+package org.json4s.reflect
+
+import org.json4s.reflect.DescriptorsSpec.{Company, Citizen, Person}
+import org.specs2.mutable.Specification
+
+object DescriptorsSpec {
+  case class Person(firstName: String, lastName: String) {
+    def this(age: Int) = this("John", "Doe")
+  }
+  object Person {
+    def apply(email: String) = new Person("Russell", "Westbrook")
+  }
+
+  case class Citizen(firstName: String, lastName: String, id: String) {
+    def this(firstName: String, lastName: String, idAsANumber: Int) = this(firstName, lastName, idAsANumber.toString)
+  }
+
+  case class Company(name: String, industry: String = "IT", ceo: Person = new Person("My", "Self"), yearFounded: Int) {
+    def this(name: String, industry: String, ceo: Person) {
```

Comment:
```scala
def this(name: String, industry: String, ceo: Person) = {
```

Shouldn't there be an '=' between the method definition and the code block?
(2.13.3 will cause a compile error)

Reply:
I think this is what Travis build says too.